src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -395,7 +395,15 @@ Apply to thead to make headers not bold.
     border: 0;
 }
 
-/* Code to morph navbar to hamburger at tablet size (992px) instead of phone size (768px) */ 
+/*
+Used for underlining span elements with extra information as titles.
```

Comment:
Let's leave this comment out because the rule is rather simple and the comment actually relates to code elsewhere (and thus risk getting out of sync).